REQUIRE

```diff
@@ -1,7 +1,8 @@
-julia 0.6
+julia 0.7-alpha
 Compat 0.17
-DataValues
 PooledArrays 0.2.0
+Nullables
+Dates
```

Comment:
You don't want `Dates` here; it belongs in the `Project.toml` for 0.7, but shouldn't be in the 0.6 REQUIRE